src/App.js

```diff
@@ -11,9 +11,90 @@ import HugeAss from "./Pages/HugeAss/HugeAss";
 import Footer from "./Components/Footer/Footer";
 import Episodes from "./Pages/Episodes/Episodes";
 import Playback from "./Components/Playback/Playback";
+import axios from "axios";
+
+var XMLParser = require("react-xml-parser");
+
+function convertUnicode(myString) {
+    let convertedString = myString.replaceAll("%20", " ");
+    if (convertedString.length > 55) {
+        return convertedString.slice(0, 54) + "...";
+    } else return convertedString;
+}
+
+function stringChoppy(mystring) {
+    let slicedString = mystring.slice(0, mystring.length - 2);
+
+    return slicedString;
+}
+
+function convertDate(date) {
+    let dateArray = date.split(" ");
+    let end = "";
+    let day = "";
+    let digitOne = Number(dateArray[1][0]);
+    let digitTwo = Number(dateArray[1][1]);
+
+    digitOne === 0 ? (day = digitTwo) : (day = Number(dateArray[1]));
+
+    if (day === 1 || day === 21) {
+        end = "st";
+    } else if (day === 2 || day === 22) {
+        end = "nd";
+    } else if (day === 3 || day === 23) {
+        end = "rd";
+    } else end = "th";
+
+    return `${dateArray[2]} ` + `${day + end}, ` + dateArray[3];
```

Comment:
netlify is mad about this concatonation.
Here is a page with the lint rule and different ways to go about fixing this issue
https://eslint.org/docs/latest/rules/no-useless-concat